### DISCUSSION TITLE
No Sway session in login manager (ubuntu noble, fresh install 3.2 beta 2)

### DISCUSSION THREAD

**Discussion by sedrik** [regolith-linux/regolith-desktop#1034] [Q&A]
Hi

Using the 3.2 Beta 2 release I get no Regolith sway session to login to after a reboot.

Installing and testing with the X session I get a option for it in the login manager

```sh
cat /etc/regolith/version
REGOLITH_VERSION=3.2
```

Not sure what information would be useful to debug this but I will happily share it :)

**Comment by kgilmer** [comment]
Hi @sedrik , what does `apt list --installed | grep sway-regolith` say?

**Comment by sedrik** [comment]
```bash
❯ apt list --installed | grep sway-regolith

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

sway-regolith/unknown,now 1.9-16-1regolith-noble amd64 [installed,automatic]
```

**Comment by iinuwa** [comment]
GDM might be forcing X11 instead of Wayland if you're using NVIDIA drivers.

After logging in, you can use this command to see whether you're using X11 or Wayland.
```shell
loginctl show-session (loginctl list-sessions | grep $USER | awk '{print $1}') --property=Type
```

If you are using NVIDIA drivers with GDM, you need to make sure that you fulfill the prerequisites for GDM usage. For example, from the Arch Wiki: https://wiki.archlinux.org/title/GDM#Wayland_and_the_proprietary_NVIDIA_driver.

**Reply by sedrik** [reply]
> GDM might be forcing X11 instead of Wayland if you're using NVIDIA drivers.
> 
> After logging in, you can use this command to see whether you're using X11 or Wayland.
> 
> ```shell
> loginctl show-session (loginctl list-sessions | grep $USER | awk '{print $1}') --property=Type
> ```
> 
> If you are using NVIDIA drivers with GDM, you need to make sure that you fulfill the prerequisites for GDM usage. For example, from the Arch Wiki: https://wiki.archlinux.org/title/GDM#Wayland_and_the_proprietary_NVIDIA_driver.

Thanks for the suggestion.

I don't have a Nvidia card in this machine so this is not the cause. Also I know that I am not running Wayland as the way option does not appear.

**Reply by kgilmer** [reply]
Do you have the sway session package installed?

```
$ apt list --installed | grep regolith-session
...
regolith-session-sway/unknown,now 1.1.12-1regolith-testing amd64 [installed]
```

**Reply by sedrik** [reply]
Yes, sorry for the extremely late reply, I had saved the notification to respond when I was at the particular computer and then forgot about it for far to long. This is still an issue

```
✦ ❯ apt list --installed | grep regolith-session

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

regolith-session-common/unknown,now 1.1.12-1regolith-noble amd64 [installed,automatic]
regolith-session-flashback-ext/unknown,now 1.1.12-1regolith-noble amd64 [installed,automatic]
regolith-session-flashback/unknown,now 1.1.12-1regolith-noble amd64 [installed]
regolith-session-sway/unknown,now 1.1.12-1regolith-noble amd64 [installed]
```

**Comment by Samiisd** [comment]
exactly the same issue for me, on amd integrated gpu
